pyproject.toml

```diff
@@ -5,7 +5,6 @@ requires = [
     # 70.1.0: min version for integrated bdist_wheel command from wheel package
     # 77.0.0: min version for SPDX expression support for project.license
     "setuptools>=70.1.0",
-    "wheel",
```

Comment:
Are we sure, we don't test with build isolation... I think this might have been an issue with uv, but I forget.

Reply:
The previous PR in the stack:

- #157783

replaces `wheel.bdist_wheel.bdist_wheel` with `setuptools.command.bdist_wheel.bdist_wheel`.